<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MappingSendDialog</class>
 <widget class="QWidget" name="AddStreamDialog">

  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>

  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add custom transformations</string>
  </property>

  <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
            </property>



            <item>
                <layout class="QFormLayout" name="form">
                
                    <item row="0" column="0">
                        <widget class="QLabel" name="transform_label">
                            <property name="text">
                            <string>Transformation</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QComboBox" name="transformDropdown"/>
                    </item>

                    <item row="1" column="0">
                        <widget class="QLabel" name="layers_label">
                            <property name="text">
                            <string>Layer</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QComboBox" name="layerDropdown"/>
                    </item>

                    
                    <item row="2" column="0">
                        <widget class="QLabel" name="attr_label">
                            <property name="text">
                            <string>Attribute</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QComboBox" name="attrDropdown"/>
                    </item>


                    <item row="3" column="1">
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item >
                                <widget class="QPushButton" name="addTransform">
                                    <property name="text">
                                    <string>+</string>
                                    </property>
                                </widget>
                            </item>
                            
                            <item >
                                <widget class="QPushButton" name="removeTransform">
                                    <property name="text">
                                    <string>-</string>
                                    </property>
                                </widget>
                            </item>
                            
                        </layout>
                    </item>


                    <item row="4" column="1">
                        <widget class="QListWidget" name="transformationsList"/>
                        <property name="viewMode">
                        <enum>QListView::ListMode</enum>
                        </property>
                    </item>
                    
                    <item row="5" column="1">
                        <layout class="QHBoxLayout" name="horizontalLayout">
                        </layout>
                    </item>

                    
                    <item row="6" column="0">
                        <widget class="QLabel" name="elevation_label">
                            <property name="text">
                            <string>Elevation layer</string>
                            </property>
                        </widget>
                    </item>
                    <item row="6" column="1">
                        <widget class="QComboBox" name="elevationLayerDropdown"/>
                    </item>


                    <item row="7" column="1">
                        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,1">
                            <item>
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Expanding</enum>
                                    </property>
                                </spacer>
                            </item>
                            <item >
                                <widget class="QPushButton" name="more_info">
                                    <property name="text">
                                    <string>More info</string>
                                    </property>
                                </widget>
                            </item >
                            <item >
                                <widget class="QPushButton" name="dialog_button">
                                    <property name="text">
                                    <string>Ok</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>

                    
                </layout>
            </item>



        </layout>
    </item>
  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
